### PR TITLE
Misc code cleanup

### PR DESCRIPTION
**Pull request by jiangliu**
Misc code cleanup

### CODE REVIEW

File: storage/src/meta/toc.rs [hunk 1]
```diff
@@ -335,56 +333,54 @@ impl TocEntryList {
...
         if entry_type != EntryType::Regular {
-            return Err(anyhow!("Tar entry type for ToC is not a regular file"));
+            return Err(eother!("Tar entry type for ToC is not a regular file"));
         }
         let entry_size = header
             .entry_size()
-            .map_err(|_| anyhow!("failed to get entry size from tar header"))?;
+            .map_err(|_| eother!("failed to get entry size from tar header"))?;
         if entry_size != size as u64 {
-            return Err(anyhow!(
```
Comment (changweige) [inline storage/src/meta/toc.rs:370]: Why not use `anyhow`

File: src/bin/nydus-image/core/feature.rs [hunk 2]
```diff
@@ -45,6 +45,7 @@ impl TryFrom<&str> for Feature {
     fn try_from(f: &str) -> std::result::Result<Self, Self::Error> {
         match f {
             "blob_toc" => Ok(Self::BlobToc),
```
Comment (imeoer) [inline src/bin/nydus-image/core/feature.rs:47]: Snapshotter converter has changed to `blob-toc`, we can remove it safely.

File: storage/src/meta/toc.rs (hunk 1 above)
Comment (imeoer) [reply storage/src/meta/toc.rs:370]: It seems the `anyhow` crate only be introduced in `nydus-image`.

File: src/bin/nydus-image/core/feature.rs (hunk 2 above)
Comment (imeoer) [reply src/bin/nydus-image/core/feature.rs:47]: @jiangliu It seems the `blob_toc` still be checked in `src/bin/nydus-image/main.rs`.

Comment (jiangliu) [reply src/bin/nydus-image/core/feature.rs]: Fixed by changing to "blob-toc".

**Pull request merged**